spec/models/team_performance_spec.rb

```diff
@@ -66,14 +59,6 @@ def create_all_teams
         end
       end
 
-      context 'for a team with older activities and older feedback' do
-        subject { TeamPerformance.new(team_both_outdated)}
-
-        it "signals green" do
-          expect(subject.evaluation).to eq(:green)
-        end
-      end
-
```

Comment:
I am not sure if all this specs should be removed. I didn't dive deep, but I'd expect that we only want to remove the specs that were added  here: https://github.com/rails-girls-summer-of-code/rgsoc-teams/commit/28b932e4a4b7ad3cd8d7ca54a124dcb96dc4cb67#diff-1144034ef8126927cabe1a2a9195b521
Could you double check if we aren't losing usefull specs here?